### PR TITLE
Create a view for a child list

### PR DESCRIPTION
**Pull request by gik0geck0**
Change Users table to be case-insensitive & snake_case
Configure sequelize to no longer quote entity names (columns/attributes & table names in postgresql)
Link & consume Contact / RecordType data
Add a nav bar to switch between pages

### CODE REVIEW

File: config/passport.js [hunk 1]
```diff
@@ -3,26 +3,15 @@ var LocalStrategy    = require('passport-local').Strategy;
 var FacebookStrategy = require('passport-facebook').Strategy;
 var GoogleStrategy   = require('passport-google-oauth').OAuth2Strategy;
 
-// load up the user model
-var configDB = require('./database.js');
```
Comment (gik0geck0) [inline config/passport.js:7]: I moved all this into data.js. Basically, if you want to look-up data, require data.js, and use the objects/methods it exports. This way, the model only gets loaded once.

File: server.js [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 // set up ======================================================================
 // get all the tools we need
+var env = require('node-env-file');
+env(__dirname + '/.env');
```
Comment (gik0geck0) [inline server.js:6]: Loads your local .env file, so that you can locally override the DB/etc vars

File: app/models/local_user.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+		google_email			: DataTypes.STRING,
+		google_name				: DataTypes.STRING
+	},
+	{
+		underscored: true,
+		underscoredAll: true,
+		classMethods: {
+			generateHash : function(password) {
+				var hash = bcrypt.hashSync(password, bcrypt.genSaltSync(8), null);
+				console.log("Hashing " + password + " -> " + hash);
```
Comment (roby-handinhand) [inline app/models/local_user.js]: Should we be logging an unencrypted password to the console?

File: app/models/local_user.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+				return hash;
+			},
+		},
+		instanceMethods: {
+			validPassword : function(password) {
+				console.log("Comparing passwords: (" + password + "," + this.local_password + ")");
```
Comment (roby-handinhand) [inline app/models/local_user.js]: ditto

**Review by roby-handinhand — APPROVED** [review]

File: app/models/local_user.js (hunk 3 above)
Comment (gik0geck0) [reply app/models/local_user.js]: Hah. No. I'll remove this.

File: app/models/local_user.js (hunk 4 above)
Comment (gik0geck0) [reply app/models/local_user.js]: Hah. No. I'll remove this.

**Pull request merged**